wayland.c

```diff
@@ -144,7 +158,7 @@ static void touch_handle_up(void *data, struct wl_touch *wl_touch,
 	wl_list_for_each(notif, &state->notifications, link) {
 		if (hotspot_at(&notif->hotspot, seat->touch.pts[id].x, seat->touch.pts[id].y)) {
 			struct mako_surface *surface = notif->surface;
-			notification_handle_touch(notif, &ctx);
+			notification_handle_touch(notif, &ctx, time - seat->touch.pts[id].time);
```

Comment:
If we already have a timer, I don't think we need to also handle the timeout here?